### DISCUSSION TITLE
How do i uninstall EverythingPowerToys?

### DISCUSSION THREAD

**Discussion by Tim-Odermatt** [lin-ycv/EverythingPowerToys#114] [Q&A]
I have downloaded EverythingPowerToys on a seperate Comuter, I now dont need it anymore.

How do i uninstall EverythingPowerToys?

**Comment by lin-ycv** [comment]
Assuming you're using one of the recent versions, just delete this folder `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins\Everything`
